homeassistant/components/elgato/light.py

```diff
@@ -110,23 +109,26 @@ def is_on(self) -> bool:
 
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn off the light."""
-        await self.async_turn_on(on=False)
+        try:
+            await self.elgato.light(on=False)
+        except ElgatoError:
```

Comment:
suggestion use “as variable” and log the content of the variable that gives a more precise logging of the problem instead of just saying error

Reply:
Sure, but not related to this PR.

Reply:
but this PR adds the try/exception so why not add it user friendly.

Reply:
This PR moves it from the turn_on method to adhere to typing. I'd decorate it if I truly was refactoring.